src/containerapp/azext_containerapp/tests/latest/test_containerapp_env_httprouteconfig.py

```diff
@@ -0,0 +1,116 @@
+        ])
+
+        route_name = "route1"
+
+        self.cmd("az containerapp env http-route-config create -g {} -n {} -r {} --yaml {}".format(resource_group, env_name, route_name, httprouteconfig1_file_name), checks=[
+            JMESPathCheck('properties.provisioningState', "SucceededWithErrors"),
+            JMESPathCheck('properties.provisioningErrors[0].message', "Unable to retrieve container app app1 from cluster"),
+            # Not deployed yet
+            # JMESPathCheck('properties.rules[0].description', "rule 1"),
+            JMESPathCheck('properties.rules[0].routes[0].match.prefix', "/1"),
+            JMESPathCheck('properties.rules[0].routes[0].action.prefixRewrite', "/"),
+            JMESPathCheck('properties.rules[0].targets[0].containerApp', "app1"),
+        ])
+
+        self.cmd("az containerapp env http-route-config show -g {} -n {} -r {}".format(resource_group, env_name, route_name), checks=[
+            JMESPathCheck('properties.provisioningState', "SucceededWithErrors"),
+            JMESPathCheck('properties.provisioningErrors[0].message', "Unable to retrieve container app app1 from cluster"),
+            # Not deployed yet
+            # JMESPathCheck('properties.rules[0].description', "rule 1"),
+            JMESPathCheck('properties.rules[0].routes[0].match.prefix', "/1"),
+            JMESPathCheck('properties.rules[0].routes[0].action.prefixRewrite', "/"),
+            JMESPathCheck('properties.rules[0].targets[0].containerApp', "app1"),
+        ])
+
+        self.cmd("az containerapp env http-route-config list -g {} -n {}".format(resource_group, env_name), checks=[
+            JMESPathCheck('[0].properties.provisioningState', "SucceededWithErrors"),
+            JMESPathCheck('[0].properties.provisioningErrors[0].message', "Unable to retrieve container app app1 from cluster"),
+            # Not deployed yet
+            # JMESPathCheck('[0].properties.rules[0].description', "rule 1"),
+            JMESPathCheck('[0].properties.rules[0].routes[0].match.prefix', "/1"),
+            JMESPathCheck('[0].properties.rules[0].routes[0].action.prefixRewrite', "/"),
+            JMESPathCheck('[0].properties.rules[0].targets[0].containerApp', "app1"),
+        ])
+
+        self.cmd("az containerapp env http-route-config update -g {} -n {} -r {} --yaml {}".format(resource_group, env_name, route_name, httprouteconfig2_file_name), checks=[
+            JMESPathCheck('properties.provisioningState', "SucceededWithErrors"),
+            JMESPathCheck('properties.provisioningErrors[0].message', "Unable to retrieve container app app2 from cluster"),
+            # Not deployed yet
+            # JMESPathCheck('properties.rules[0].description', "rule 2"),
+            JMESPathCheck('properties.rules[0].routes[0].match.prefix', "/2"),
+            JMESPathCheck('properties.rules[0].routes[0].action.prefixRewrite', "/"),
+            JMESPathCheck('properties.rules[0].targets[0].containerApp', "app2"),
+        ])
+
```

Comment:
It seems that all 4 test cases are failed case. Would you pls help to add some successful test case, for example, the `app1` exists in the env, what will happen?